app/presenters/account_relationships_presenter.rb

```diff
@@ -36,6 +38,37 @@ def initialize(account_ids, current_account_id, **options)
 
   private
 
+  def domain_blocking_map
+    target_domains = @accounts.pluck(:domain).compact.uniq
```

Comment:
Minor! - `target_domains` could be a memoized private method and avoid the local var.